ldk/javascript/examples/self-test-loop/src/selfTest.ts

```diff
@@ -275,6 +280,12 @@ const testConfig: { [key: string]: TestGroup } = {
       20000,
       `Did the whisper update correctly?`,
     ),
+    new LoopTest(
+      'Whisper Update - Automated OnChange',
+      whisperStateOnChange,
+      20000,
```

Comment:
If no action needed, do we need to have it 20sec? in case of failure it feels like a long timeout.